spec/unit/shoryuken/error_handler_spec.rb

```diff
@@ -0,0 +1,53 @@
+require 'spec_helper'
+require 'airbrake/shoryuken/error_handler'
+
+module Airbrake
+  ##
+  # Provides tests for the Shoryuken integration.
+  module Shoryuken
+    RSpec.describe ErrorHandler do
+      let(:error) { AirbrakeTestError.new('shoryuken error') }
+      let(:body) { { message: 'message' } }
+      let(:notice) { double 'Notice' }
+
+      context "when there's an error" do
+        it 'notifies' do
+          expect(Airbrake).to receive(:build_notice).with(error, body: body).
+            and_return(notice)
```

Comment:
We use Webmock to test the library, so please avoid mocks at all costs :)

This is the equivalent of this test: https://github.com/airbrake/airbrake/blob/c45bd6e0b4955dd0a0ad7fa87590383cc65fd8a4/spec/unit/sidekiq/error_handler_spec.rb#L29